TurkishFilter/sections/specific.txt

```diff
@@ -2889,7 +2891,7 @@ duzcepostasi.com##td[style$="570px;"] > table[width="570"] + img[src*="/resim/up
 ||fullhdfilmizlesene.org/rkx/
 ||163.172.18.41/apps/frag.php
 fullhdfilmizlesene.org##div[style*="width:980px;height:91px;"]
-fullhdfilmizlersen.com,dizilab.pw,dizipas.net###sticky
+filmizleplay.com,fullhdfilmizlersen.com,dizilab.pw,dizipas.net###sticky
```

Comment:
check GIF's size - 2.5 MB
https://chrome.google.com/webstore/detail/image-size-info/oihdhfbfoagfkpcncinlbhfdgpegcigf

Reply:
ok, but in any case we need to remove leftover